tests/rptest/services/openmessaging_benchmark.py

```diff
@@ -34,11 +34,14 @@ class OpenMessagingBenchmarkWorkers(Service):
         }
     }
 
-    def __init__(self, ctx, num_workers=3, nodes: Optional[list] = None):
+    def __init__(self, ctx, num_workers=None, nodes: Optional[list] = None):
         """
-        :param num_workers: allocate this many nodes as workers
-        :param nodes: use these pre-allocated nodes as workers
+        :param num_workers: allocate this many nodes as workers (mutually exclusive with `nodes`)
+        :param nodes: use these pre-allocated nodes as workers (mutually exclusive with `num_workers`)
         """
+        if nodes is None and num_workers is None:
+            num_workers = 3
```

Comment:
dunno if mutual exclusion the param description should be verified with something like an else case asserting that both nodes and num_workers aren't non-None